test/cli_crud_test.go

```diff
@@ -83,97 +66,3 @@ func TestSimpleFnTriggerUpdateCycle(t *testing.T) {
 	h.Fn("update", "trigger", appName1, funcName1, "mytrigger", "--annotation", `"val1='["val2"]'"`).AssertSuccess()
 	h.Fn("config", "trigger", appName1, funcName1, "mytrigger", "confA", "valB").AssertSuccess()
 }
-func TestRemovingRouteAnnotation(t *testing.T) {
-	t.Parallel()
-
-	h := testharness.Create(t)
-	defer h.Cleanup()
-	appName1 := h.NewAppName()
-	h.Fn("create", "route", appName1, "myroute", "foo/duffimage:0.0.1", "--annotation", "test=1").AssertSuccess()
-	h.Fn("inspect", "route", appName1, "myroute").AssertSuccess().AssertStdoutContainsJSON([]string{"annotations", "test"}, 1.0)
-	h.Fn("update", "route", appName1, "myroute", "foo/duffimage:0.0.1", "--annotation", `test=""`).AssertSuccess()
-	h.Fn("inspect", "route", appName1, "myroute").AssertSuccess().AssertStdoutMissingJSONPath([]string{"annotations", "test"})
-}
-
-func TestInvalidAnnotationValue(t *testing.T) {
-	t.Parallel()
-
-	h := testharness.Create(t)
-	defer h.Cleanup()
-	appName1 := h.NewAppName()
-
-	// The route should still be created, but without the invalid annotation
-	h.Fn("create", "route", appName1, "myroute", "foo/duffimage:0.0.1", "--annotation", "test=value").AssertSuccess().AssertStderrContains("Unable to parse annotation value 'value'. Annotations values must be valid JSON strings.")
-	h.Fn("inspect", "route", appName1, "myroute").AssertSuccess().AssertStdoutMissingJSONPath([]string{"annotations", "test"})
-}
-
-func TestRouteUpdateValues(t *testing.T) {
```

Comment:
We don't test updating values on any other object. Let's migrate this test to fns.